### ISSUE TITLE
Update error when languages directory is missing

### ISSUE THREAD

**Issue by slcr** [uploadcare/uploadcare-wordpress#23]
when updating from `2.4.1` to `2.5.0` i had no `/wp-content/languages/` directory, because it was unneeded yet. the plugin update showed an error about a missing languages directory. it still installed the `2.5.0` version, but showed some errors on the settings-page.

```
Warning: in_array() expects parameter 2 to be array, boolean given in /www/htdocs/xxx/wp-content/plugins/uploadcare/inc/utils.php on line 166

Warning: array_search() expects parameter 2 to be array, boolean given in /www/htdocs/xxx/wp-content/plugins/uploadcare/inc/utils.php on line 171

Warning: implode(): Invalid arguments passed in /www/htdocs/xxx/wp-content/plugins/uploadcare/inc/utils.php on line 182
```

it's seems to be fixed with updating the settings, still wanted to let you know about it.

**Comment by ZNick1982** [comment]
Hi @slcr !
Thanks for the issue.
I will check and fix this if necessary.

**Comment by ZNick1982** [comment]
Hi @slcr .
I checked plugin but its still required language directory because of Wordpress API, Can you create it and just keep it empty?
I fixed warnings in utils.php.
All fixes included in  v2.5.1
Also updated on Wordpress website.

**Comment by ZNick1982** [comment]
Closed due to no comments.
